Dockerfile

```diff
@@ -1,13 +1,23 @@
+FROM registry.access.redhat.com/ubi8:8.4 as build
 
-FROM registry.access.redhat.com/ubi8/nodejs-14:1
-
+RUN  dnf module install --nodocs -y nodejs:14 python39 --setopt=install_weak_deps=0 --disableplugin=subscription-manager \
+    && dnf install --nodocs -y make gcc gcc-c++  --setopt=install_weak_deps=0 --disableplugin=subscription-manager \
+    && dnf clean all --disableplugin=subscription-manager
+    
 RUN mkdir -p /opt/app-root/src
 WORKDIR /opt/app-root/src
 COPY package.json /opt/app-root/src
-RUN npm install --only=prod
+RUN npm install --no-audit --no-update-notifier --no-fund --production
 COPY . .
 
-ENV NODE_ENV production
+## Release image
+FROM registry.access.redhat.com/ubi8/nodejs-14-minimal:latest
```

Comment:
Can you pin the image version here? I tested this file and got this error in dockerlint:
```
Dockerfile found at: ./Dockerfile
Linting Dockerfile
./Dockerfile:14 DL3007 Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag
```

Reply:
Yeah - hate that linter warning.